RUNNING.md

```diff
@@ -36,16 +36,16 @@ Compile QEMU by running the following commands from the root directory:
 mkdir build
 cd build
 # On Intel Macs
-../configure --target-list=arm-softmmu --extra-cflags=-I/usr/local/opt/openssl@3/include --extra-ldflags='-L/usr/local/opt/openssl@3/lib -lcrypto'
+./configure --target-list=arm-softmmu --extra-cflags=-I/usr/local/opt/openssl@3/include --extra-ldflags='-L/usr/local/opt/openssl@3/lib -lcrypto'
```

Comment:
no, the `build` dir is empty , it is created 2 lines above

Reply:
```
mkdir build
cd build
```
can be removed when using `./configure` since build directory is created automatically

Reply:
Shouldn't rely on legacy and undocumented mechanisms, the official qemu documentation recommends to create the `build` folder manually: https://www.qemu.org/docs/master/devel/build-system.html

Reply:
ok 👌

Reply:
Agreed, I think it is standard to create the `build` directory yourself. Can you adjust the PR accordingly?

Reply:
Okay I will modify it thank you for the feedback!